roles/docker/tasks/main.yml

```diff
@@ -36,3 +36,39 @@
   when:
     - l_use_crio
     - openshift_docker_is_node_or_master | bool
+
+- when:
+  - l_use_crio
+  - l_use_system_container
```

Comment:
I believe this should be removed. cri-o is always a system container, but cri-o lives along side rpm or system container installs.